### DISCUSSION TITLE
Welcome! Introduce yourself :)

### DISCUSSION THREAD

**Discussion by soerenthomsen** [OceanGlidersCommunity/Oxygen_SOP#2] [General]
Dear OceanGliders Community, 

here we will discuss everything relevant to oxygen measurements on gliders. 

Both experts and beginners are welcome to contribute to the SOP. Feel free to introduce yourself quickly i.e. let us know what kind of glider / oxygen sensor combination you are using. 

Best 
Soeren

**Comment by soerenthomsen** [comment]
Hi everyone,

I am Soeren Thomsen, Physical Oceanographer. Right now I work 50 % as OceanGliders.org Best Practice coordinator.

I have been working with Slocum gliders since around 2011. 
For my PhD (2012-2016) and PostDoc at GEOMAR (2017-2018) I designed and carried out two glider fleet experiments off Peru (https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2015JC010878, https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2016GL070548) and one Mauritania (https://bg.copernicus.org/articles/16/979/2019/). 
Oxygen measurements where a key part of these experiments. Luckily I got support from @hahn-johannes and @gkrahmann for the oxygen calibration.

I am a big fan of open science. So all related datasets are quality controlled open access available on Pangaea (https://doi.pangaea.de/10.1594/PANGAEA.860727, https://doi.pangaea.de/10.1594/PANGAEA.860480) thanks to G. Krahmann and the data management team at GEOMAR 🙏. Please get in contact if you are interested in glider datasets in upwelling systems (i.e. off Peru, Mauritania, Namibia or Benguela). I can navigate you through the different available deployments available on Pangaea.

I switched to Python in 2018 mainly to analyse ROMS/CROCO model output using xarray, dask etc! So far I have little experience in collective coding but motivated to learn on the fly!

Looking forward to working with all of you on this oxygen SOP.

Best
Soeren

PS: I nominate the next one to introduce herself: @patricialg :D

**Reply by patricialg** [reply]
Hi all,
My name is Patricia and I work as Sensor Science Delivery Lead at NOC in Southampton. I'm currently working with nutrient sensors in the lab and as delivery lead for some projects like TechOceanS, AutoNuts, etc.. In these projects we develop sensors that are integrated in differnt platforms. In TechOceanS project, I'm also involved in the work related with the development of Best Practices for sensors.
I'm a marine scientist. I have extensive laboratory experience including some of the latest marine chemistry and trace analysis techniques.
From 2014-2019 I was responsible for collection and analysis of samples from the European Station for Time series in the Ocean Canary Islands (ESTOC) and the Test Site of PLOCAN in addition to the calibration and validation of sensors integrated in moorings and gliders. I used to work closely with glider pilots (I was the lab rat in a white/mostly white lab coat). 
I'm contributing (or trying to contribute) to the Glider best practices document that Soeren is coordinating. I'm helping with the Oxygen sensors section. My main interest is to help, when possible, to calibrate/validate sensors so that they will produce the best possible quality data and find the best protocol for the gold standard method to achieve this. Gliders are the future of Ocean Observation but without good sensors, they are only fancy toys, so I really believe that this initiative is very important.
I'm happy to discuss, learn from others and look together for the best approaches :)
Cheers,
Patry
PS: I nominate @tomhull  :)

**Reply by soerenthomsen** [reply]
@patricialg we try to restart the intro round for the community here: 
https://github.com/OceanGlidersCommunity/OceanGliders/discussions/1

i.e. for all repos and not specific to the oxygen one. Would you mind copying over your intro?

**Reply by patricialg** [reply]
done :)

**Comment by tomhull** [comment]
Hi everyone

I'm Tom. I'm the autonomous ocean systems principle investigator at [Cefas](https://www.cefas.co.uk/). My background is biogeochemistry, mostly oxygen and air-sea gas exchange, but I've been working with autonomous buoys for around 10 years and gliders for the last 4 under [AlterEco](https://altereco.ac.uk/). I'm lucky enough to be part of the whole process, from the workshop to the published data and science.

I set up this GitHub repository and configured the pandoc build tools for the pdf/docx output, so any problems with that feel free to @me.

@hahn-johannes

**Reply by soerenthomsen** [reply]
@tomhull we try to restart the intro round for the community here:
https://github.com/OceanGlidersCommunity/OceanGliders/discussions/1

i.e. for all repos and not specific to the oxygen one. Would you mind copying over your intro?
